### ISSUE TITLE
suggest to index IPBES documents 

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/globalbioticinteractions#911]
IPBES is a the Intergovernmental Science-Policy Platform on Biodiversity and Ecosystem Services (IPBES) is an intergovernmental organization established in 2012 to improve the interface between science and policy on issues of biodiversity and ecosystem services.

![image](https://github.com/globalbioticinteractions/globalbioticinteractions/assets/1084872/26a52e2e-abdb-4f91-9d37-9efa6591f56d)

**Comment by jhpoelen** [comment]
fyi @n8upham @kephelps @arw36

**Comment by jhpoelen** [comment]
As discussed with Jorrit Poelen (GloBI), Nathalia de la Calzada, Theo Alves (Data for Good), Maral Dadivar, Rainer Krug, Aidin Niamir (IPBES) and Renske Gudde (IPBES)

**Comment by jhpoelen** [comment]
See associated search results _Paguma larvata_  at https://www.globalbioticinteractions.org/?interactionType=interactsWith&sourceTaxon=Paguma%20larvata or screenshot made on 2023-07-12 below.

![image](https://github.com/globalbioticinteractions/globalbioticinteractions/assets/1084872/cd6b2e60-7d9e-4a21-9468-c4c0b50e249c)

**Comment by jhpoelen** [comment]
@n8upham tracked the provenance of the figure claim derived from:

Huang, C. et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395,
497-506, doi:10.1016/S0140-6736(20)30183-5 (2020).

**Comment by jhpoelen** [comment]
With @Daniel-Mietchen , we transcribed some pollinator interaction claims in:

IPBES. (2016). The assessment report of the Intergovernmental Science-Policy Platform on Biodiversity and Ecosystem Services on pollinators, pollination and food production. Table 2.4.3 p88 Zenodo. https://doi.org/10.5281/zenodo.3402857

via 

index configuration at 

https://github.com/globalbioticinteractions/ipbes-pollinator-2016

**Comment by jhpoelen** [comment]
@qgroom would you happen to have the raw data associated with the 2016 IPBES pollinator assessment?

**Comment by Daniel-Mietchen** [comment]
[Here](https://scholia.toolforge.org/work/Q120718408) is the skeleton of a Wikidata profile for the report, with the only data points in there stemming from the references underlying table 2.4.3:

![Screenshot 2023-07-20 at 01-48-33 The assessment report of the Intergovernmental Science-Policy Platform on Biodiversity and Ecosystem Services on pollinators pollination and food production - Scholia](https://github.com/globalbioticinteractions/globalbioticinteractions/assets/465923/bb24a8f8-69ca-4fca-88db-750b3fe2dffa)

**Comment by Daniel-Mietchen** [comment]
Some of the cited references have remarkable embargo periods:

- Thalmann and Dorn (1990) is embargoed until 2050
![image](https://github.com/globalbioticinteractions/globalbioticinteractions/assets/465923/057702ec-30b3-4f22-b537-d986b967279d)
(note that IPBES lists ```Thalmann``` as ```Thalann```)

- Batra (1994) is US government work and thus supposedly in the public domain, yet I could not find a public copy, and the copy I did find was embargoed until - wait for it - **January 01, 4500**:
![image](https://github.com/globalbioticinteractions/globalbioticinteractions/assets/465923/a6d4b5d0-dd77-4f80-a3ed-7f97da4f71ee)

**Comment by qgroom** [comment]
> @qgroom would you happen to have the raw data associated with the 2016 IPBES pollinator assessment?

Yes, I'm quite curious. I think @mtrekels would be too.

**Comment by jhpoelen** [comment]
@qgroom good to hear you are curious! And, would you happen to know anyone that may have access to the raw data associated with the 2016 IPBES pollinator assessment ?

**Comment by qgroom** [comment]
Ah! I miss read your post. No, I have no idea. At that time the IPBES data team were in Korea, but apparently thinks did not go that well. If there are no links in the report then I think the sources are lost

**Comment by myrmoteras** [comment]
I wrote at that time a review for the pollinator assessment, and one of the points was exactly that they need to assure that the data is available, not least for the next round of reviews.

I also attended at IPBES meeting here in Switzerland where I raised the points. But they did not understand and said, there are DOIs for the articles, and did not understand that access to the extracted data used for further analysis is needed. 

After that they changed to Senkenberg. At the same time, there is no way the IPBES can assert a power and the fiercly independent rewiew team, other than to recommend. But even that seems to have to go through a review process.

**Comment by qgroom** [comment]
Yes, the situation has changed for current assessments. Each figure is meant to have a data management plan and details of the resources used. The IAS assessment is going to be released in September and it will be interesting to see how well this has worked in practice. 
IPBES assessments are written by a global team of authors and for many data management plans and Open Science will be new concepts. So I feel a lot of progress has been made since the pollination assessment.
